ansible/playbooks/roles/encoder-master/tasks/nfs.yml

```diff
@@ -4,7 +4,7 @@
     create: true
     dest: /etc/exports
     block: |
-      /var/lib/sreview	10.0.0.0/8(rw,sync)
+      /var/lib/sreview	10.0.0.0/8(rw,sync)  # noqa 203
```

Comment:
Hrm, now that I think about it, does `/etc/exports` require a hard tab here?

Reply:
I always to tabs in it

Reply:
do*